packages/core/src/Provider/ChainHistoryProvider/types.ts

```diff
@@ -4,8 +4,8 @@ export type TransactionsByAddressesArgs = {
   addresses: Cardano.Address[];
   sinceBlock?: Cardano.BlockNo;
 };
-export type TransactionsByHashesArgs = Cardano.TransactionId[];
-export type BlocksByHashesArgs = Cardano.BlockId[];
+export type TransactionsByHashesArgs = { hashes: Cardano.TransactionId[] };
```

Comment:
This highlights a slight awkwardness with these interface names. Given we're breaking the API of these methods here, I propose we also change `transactionsByHashes` to `transactionsByIds` etc first, then this commit can use a more aligned:

```suggestion
export type TransactionsByIdsArgs = { ids: Cardano.TransactionId[] };
```

Reply:
We should unify all references in the SDK. I've seen both `id` and `hash` being used in the wild, so we just need to pick one and refactor entire SDK. Converting the type to `Cardano.TransactionHash` would be ok too. I'm not sure which one I like better, maybe `hash` as it's more specific :thinking:

Reply:
Let's stick with `id`, as it's what the ledger code uses: https://github.com/input-output-hk/cardano-ledger/search?p=2&q=transaction.Id